client/components/domains/register-domain-step/index.jsx

```diff
@@ -527,6 +585,7 @@ class RegisterDomainStep extends React.Component {
 			tld_weight_overrides: null,
 			vendor: 'wpcom',
 			vertical: this.props.surveyVertical,
+			...this.getSetFilters(),
```

Comment:
When we send queries to the backend, we use `snake_case`, not `camelCase` - it'd be great for consistency to stick to that convention. lodash has a [nifty utility function called, well, `snakeCase`](https://lodash.com/docs/#snakeCase) that should make this easy.

Reply:
Fixed!